app/frontend/src/App.tsx

```diff
@@ -7,14 +7,22 @@ import Navigation from "./components/Navigation";
 import PageTitle from "./components/PageTitle";
 import { theme } from "./theme";
 
+const useStyles = makeStyles(theme => ({
+  offset: theme.mixins.toolbar,
+}))
+
 function App() {
+  const classes = useStyles();
   return (
     <Router>
       <ThemeProvider theme={theme}>
         <CssBaseline />
+        <Navigation />
+        <Hidden smDown>
+          <div className={classes.offset} />
+        </Hidden>
         <Container maxWidth="md">
           <PageTitle>Couchers</PageTitle>
```

Comment:
Maybe we don't need this in desktop

```suggestion
          <Hidden mdUp>
            <PageTitle>Couchers</PageTitle>
          </Hidden>
```

Reply:
But in that case, there needs to be a bit of padding above it.

Reply:
Maybe it is nice to put this bar in Storybook? Look at the Avatar and the messages tab how it can be done.